azure-pipelines.yml

```
@@ -28,22 +28,22 @@ stages:
           matrix:
             Python36Ubuntu:
               imageName: 'ubuntu-latest'
-              python.version: 3.6
+              python.version: 3.7
             Python37Ubuntu:
               imageName: 'ubuntu-latest'
-              python.version: 3.7
+              python.version: 3.8
             Python38Ubuntu:
               imageName: 'ubuntu-latest'
-              python.version: 3.8
+              python.version: 3.9
             Python36macOS:
               imageName: 'macOS-latest'
-              python.version: 3.6
+              python.version: 3.7
             Python37macOS:
```

The names of these matrices should be updated too.
```suggestion
            Python38macOS:
```